components/formats-gpl/src/loci/formats/in/InCellReader.java

```diff
@@ -1018,9 +1063,10 @@ else if (qName.equals("Exposure") && openImage) {
       else if (qName.equals("offset_point")) {
         String x = attributes.getValue("x");
         String y = attributes.getValue("y");
+        int index = Integer.parseInt(attributes.getValue("index"));
```

Comment:
This line fails some of the [automated test](https://ci.openmicroscopy.org/job/BIOFORMATS-DEV-merge-repository-subset/286/consoleText) with a NPE. It might be worth using `DataTools.parseInteger` and handle the case where `index == null`.